feedback/models.py

```diff
@@ -34,6 +36,10 @@ class IssueChoices(models.TextChoices):
     created = models.DateTimeField(auto_now_add=True)
     modified = models.DateTimeField(auto_now=True)
 
+    created_by = models.ForeignKey(
+        CustomUser, on_delete=models.CASCADE, blank=False, null=True
```

Comment:
Might be worth using the `get_user_model()` helper to refer to the User model. https://docs.djangoproject.com/en/5.1/topics/auth/customizing/#django.contrib.auth.get_user_model
``
from django.contrib.auth import get_user_model

created_by = models.ForeignKey(
        get_user_model(), on_delete=models.CASCADE, blank=False, null=True
)
``